app/Http/Controllers/LoginController.php

```diff
@@ -59,18 +64,32 @@ public function signup()
     public function postSignup(SignupRequest $request)
     {
         $input = $request->all();
-
         if (!empty($input)) {
+            $tmpPassword = $input['password'];
             $input['password'] = bcrypt($input['password']);
             $input['fullname'] = 'your fullname';
+            $input['gender'] = '1';
+            $input['image_path'] = config('constants.IMAGE_UPLOAD_PATH') . config('constants.DEFAULT_USER_IMAGE');
+            $input['role_id'] = '2';
+            if ($user = $this->userRepository->create($input)) {
+                    event(new Registered($user));
 
-            if ($this->userRepository->create($input)) {
-                \Session::flash('success_alert', __('alert.success.login'));
-
-                return redirect()->route('home.index');
+                    return view('home.account.verify', compact('user'));
             } else {
                 return redirect()->route('home.signup')->withErrors(['incorrectInfo' => __('alert.error.signup')]);
             }
         }
     }
-}
+
+    public function resend(Request $request, $id)
+    {
+        $user = $this->userRepository->find($id);
+        if ($user->hasVerifiedEmail()) {
+            return redirect()->route('home.index');
+        }
+        event(new Registered($user));
+        \Session::flash('resent', __('alert.success.resendSuccess'));
+
+        return view('home.account.verify', compact('user'));
+    }
+}
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Http/Controllers/LoginController.php</br>**Line**: 95</br>Expected 1 newline at end of file; 0 found